docs/github-connector/installation.md

```diff
@@ -0,0 +1,27 @@
+# GitHub Connector Installation
+
+- [GitHub Connector Installation](#github-connector-installation)
+  - [Installation in Kyma with Helm](#installation-in-kyma-with-helm)
+    - [Prerequisites](#prerequisites)
+    - [Steps](#steps)
+
+## Installation in Kyma with Helm
+
+### Prerequisites
+
+- Connection to Kyma cluster
+- GitHub Connector Docker image
+
+### Steps
+
+1. Go to [Kyma repository](https://github.com/kyma-project/kyma) and run script `/installation/scripts/tiller-tls.sh` to get certificates needed for using Helm commands. By default they are stored in `~/.helm` directory. After that add `--tls` flag to every Helm command to authorize and authenticate yourself
```

Comment:
```suggestion
1. Go to [Kyma repository](https://github.com/kyma-project/kyma) and run script `/installation/scripts/tiller-tls.sh` to get certificates needed for using Helm commands. By default they are stored in `~/.helm` directory. After that add the `--tls` flag to every Helm command to authorize and authenticate yourself
```

I don't like the `yourself` part. It sounds too colloquial and too general. What do you mean by that? To authenticate a user, an app? Please, specify.